diagnostic_feedback/tests/integration/test_student_bf_quiz.py

```diff
@@ -0,0 +1,118 @@
+from .base_test import StudentBaseTest
+
+
+class StudentBuzzFeedStyleTest(StudentBaseTest):
+
+    def test_for_single_group_monarch(self):
+        self.load_student_view('bf_quiz', {"mode": "standard"})
```

Comment:
@jareerahsan What is the purpose of `{"mode": "standard"}`? None of the XML templates seem to require a `mode` parameter.